pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -463,6 +463,8 @@ func convertRbacRulesToFilterConfig(
 				// Generate the policy if the service is matched to the services specified in ServiceRole.
 				rbacLog.Debugf("matched AccessRule[%d]", i)
 				permissions = append(permissions, convertToPermission(rule))
+			} else {
+				rbacLog.Debugf("Not matched to AccessRule[%d]: %s\n", i, rule.String())
```

Comment:
no need to log this, it outputs too much text and not really helpful in debugging.

Reply:
Done.